gdrive/src/uploadPdf.js

```diff
@@ -16,7 +16,8 @@ function uploadPDF(auth, file) {
   service.files.create({
     resource: {
       name: file.name,
-      mimeType: 'application/pdf'
+      mimeType: 'application/pdf',
+      parents: ['0Bx7UhUK81Pz1SE9YZ1JUR0FPbmc']
```

Comment:
Yeh, this is a specific folder in our Google Apps account. Could definitely be added to the [`google-config.json`](https://github.com/bryancross/key2pdf/blob/refactoring-to-modules/config/google-config-example.json).